pkg/services/tree/service.proto

```diff
@@ -47,6 +48,8 @@ service TreeService {
   rpc Apply (ApplyRequest) returns (ApplyResponse);
   // GetOpLog returns a stream of logged operations starting from some height.
   rpc GetOpLog(GetOpLogRequest) returns (stream GetOpLogResponse);
+  // Healthcheck is a dummy rpc to check is service available
+  rpc Healthcheck(google.protobuf.Empty) returns (google.protobuf.Empty);
```

Comment:
we usually [add](https://github.com/nspcc-dev/neofs-node/blob/2d43892fc9bfc0067689b5ef1370873c9bb5a7b1/pkg/services/tree/service.proto#L140) empty request structs explicitly for clear extensibility

/cc @fyrchik, @cthulhu-rider

Reply:
Added custom response/request.